python_codes/contour_plots.py

```diff
@@ -1,39 +1,24 @@
 #! /usr/bin/env python3
 from pylab import *
 from netCDF4 import Dataset
-import numpy as np
 
-'''Retrieving Data from the nc files'''
+# Retrieving Data from the nc files
 data = Dataset('/home/mafields/CylconeDiagnostics/data/jupiter2d-long-main.nc')
```

Comment:
You can use the following statements to separate the file name and the read action.
```
fname = '/home/mafields/CylconeDiagnostics/data/jupiter2d-long-main.nc'
data = Dataset(fname, 'r')
```
'r' standards for "read mode"